### PR TITLE
Fixed negative notification badge

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR fixes negative notification badge when deleting task.

How to reproduce:

1. Open the same page in two tabs
2. Create a new task in one of this tabs
3. Without reloading go in the second tab on the automation tab => you should see a task but no notification-badge
4. Delete task => you should see `-1` badge

### CODE REVIEW

File: Resources/public/js/components/automation-tab/main.js [hunk 1]
```diff
@@ -263,11 +268,23 @@ define([
...
                 }
             }.bind(this));
         },
 
+        incrementNotificationBadge: function() {
+            return ++this.notificationBadge;
+        },
+
+        decrementNotificationBadge: function() {
+            if (this.notificationBadge === 0) {
```
Comment (wachterjohannes) [inline Resources/public/js/components/automation-tab/main.js]: should we keep this check? /cc @danrot

Comment (danrot) [inline Resources/public/js/components/automation-tab/main.js]: Isn't that the exact fix for the problem you've described? I would only change it to `this.notificaitonBadge <= 0`, would be safer.

Comment (wachterjohannes) [inline Resources/public/js/components/automation-tab/main.js]: no that specific bug would be fixed with this https://github.com/sulu/SuluAutomationBundle/pull/6/files#diff-aaefa6ffa05231d7a9882ee4596c2ecbR96

Comment (wachterjohannes) [inline Resources/public/js/components/automation-tab/main.js]: this would prevent other bugs to display negative values.

Comment (danrot) [inline Resources/public/js/components/automation-tab/main.js]: Okay, good for me, but I would still use `<=` instead of `===`

**Comment by wachterjohannes** [comment]
@danrot fixed

**Pull request merged**